app/src/main/java/org/fossasia/susi/ai/chat/ChatPresenter.kt

```diff
@@ -78,7 +77,6 @@ class ChatPresenter(context: Context) :
     var queueExecuting = AtomicBoolean(false)
 
     override fun onAttach(chatView: IChatView) {
```

Comment:
Remove redundant functions

Reply:
Removed onAttach from every interface where it's redundant...